.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 sudo: false
 node_js:
-  - "4"
-  - "6"
+  - "10"
```

Comment:
as  we dust off these old libraries, let's switch to GitHub actions, and add automated publication/releasing, see:

https://github.com/yargs/yargs-parser/tree/master/.github/workflows

I can update the  `release-please.yml` with the publication logic, if you add `ci.yml` for testing.